rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/LocalDataTransferServer.java

```diff
@@ -174,7 +179,11 @@ public LocalServer(Configuration conf, BookKeeperFactory bookKeeperFactory)
     @Override
     public void run()
     {
-      ExecutorService threadPool = Executors.newCachedThreadPool();
+      ExecutorService threadPool = new ThreadPoolExecutor(
+              0, getLocalTransferServerMaxThreads(conf),
+              60L, TimeUnit.SECONDS,
+              new SynchronousQueue<>(),
+              threadsNamed("lds-pool-%s"));
```

Comment:
We use `lds-pool` keyword for connection pool. I guess we may rename it to something else to remove any confusion.

Reply:
nevermind, looks good. thought something else.

Reply:
will rename to `lds-worker-%s`